### DISCUSSION TITLE
Using JavaIntro.java leads to "Cannot resolve symbol 'example':10"

### DISCUSSION THREAD

**Discussion by mfeldt** [AppIntro/AppIntro#944] [Q&A] [answered]
**AppIntro Version**: 6.1.0

**Device/Android Version**: Pixel 2

**Issue details / Repro steps / Use case background**: Simply implementing the example to start working from there for a simple slider introduction.

**Your Code**: 
"""
package com.example.webviewtest1;

import android.os.Bundle;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import com.github.appintro.AppIntro;
import com.github.appintro.AppIntroFragment;
import com.github.appintro.AppIntroPageTransformerType;
import com.github.appintro.example.R;

"""

**Stack trace / LogCat**:
```
"Cannot resolve symbol 'example':10"
```

**Comment by cortinico** [comment, marked as answer]
You should be able to fix this by replacing

> import com.github.appintro.example.R;

with

> import com.example.webviewtest1.R;
